lib/cadet/accounts/ivle.ex

```diff
@@ -0,0 +1,75 @@
+defmodule Cadet.Accounts.Ivle do
```

Comment:
(Sorry I missed this during the first review) I think the module name should be `Cadet.Accounts.IVLE` https://github.com/lexmag/elixir-style-guide#camelcase-modules

Reply:
Done with 429ca34. Do you prefer `IVLETest` or `IVLE_Test`?